Equipo2Hosteleria/src/main/java/Controlador/ControladorPanelTicketFactura.java

```diff
@@ -25,31 +39,48 @@ public void mostrarPanelTicketFactura() {
 	}
 	
 	public void accionadoBottonVolverPanelTicketFactura() {
-		this.modelo.BorrarListaCompra();
+		this.modelo.getCarroCompra().borrarCarroCompra();
 		this.controlador.navegarPanelOperatividad();
 	}
 	
 	public void accionadoBottonDesconectarsePanelTicketFactura() {
-		this.modelo.BorrarListaCompra();
+		this.modelo.getCarroCompra().borrarCarroCompra();
```

Comment:
¿aquí además de borrar el carro de la compra no debería vaciar la variable Usuario 1 de la clase cuenta que se rellena al hacer login?
Revisar todas las pantallas en las que tengáis botón de desconectar